services/src/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/config/MessageConsumerConfig.java

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.grpc.speaker.config;
+
+import org.openkilda.messaging.command.CommandMessage;
+
+import com.google.common.collect.ImmutableMap;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+
+import java.util.Map;
+
+/**
+ * The Kafka message consumer configuration.
+ */
+@Configuration
+@EnableKafka
+@PropertySource("classpath:grpc-service.properties")
+public class MessageConsumerConfig {
+    /**
+     * Kafka queue poll timeout.
+     */
+    private static final int POLL_TIMEOUT = 3000;
```

Comment:
is it intended to be hard-coded constant

Reply:
Resolved. Now will be get parameter from properties.